### ISSUE TITLE
Training on a custom dataset

### ISSUE THREAD

**Issue by khasmamad99** [timy90022/One-Shot-Object-Detection#6]
Hi,
Can you please provide set of instruction for training on a custom dataset?
Thanks.

**Comment by pipi2020** [comment]
@timy90022 I am sorry 
I also neet it to train my own dataset

**Comment by bigbigdinosaur** [comment]
this one-shot method can find objects which are not in training set, so i think there is no need for so-called custom dataset

**Comment by manyaafonso** [comment]
@bigbigdinosaur not quite. One may have a specific kind of dataset.. for example containing images of apples, oranges, etc, and want to detect lemons. This would require training.

**Comment by manyaafonso** [comment]
To anyone interested, perhaps this repository could help: https://github.com/aosokin/os2d
